src/Typing.js

```diff
@@ -197,9 +197,8 @@ class Typing extends Component {
     const { isFinished } = this.state;
 
     const cursor = this.props.cursor || <Cursor className={cursorClassName} />;
-    const filled = isFinished
-      ? children
-      : replaceTreeText(children, this.state.text, cursor, hideCursor);
+
+    const filled = replaceTreeText(children, this.state.text, cursor, isFinished ? true : hideCursor)
```

Comment:
isFinished ? true : hideCursor => isFinished || hideCursor

Reply:
just double checking, you would like
```
isFinished ? true : hideCursor
```
turned into
```
isFinished || hideCursor
```
right?

Or did you want to pace an arrow function as the alternative value to the ternary? Not totally sure.

Reply:
You are correct, it was just an arrow to indicate a change, no arrow function.